src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -1021,6 +1046,38 @@ QString EnzymeTreeItem::generateEnzymeTooltip() const {
     return TOOLTIP_TAG.arg(forwardTooltip).arg(reverseTooltip);
 }
 
+QString EnzymeTreeItem::getTypeInfo() const {
+    auto type = text(Column::Type);
+    QString result;
+    static constexpr const char* UNEXPECTED_TYPE_ERROR = QT_TR_NOOP("Unexpected enzyme type: %1");
```

Comment:
You can just have a simple and more readable:
 `QString unexpectedError = tr("Unexpected enzyme type: %1")`
line instead of all that magic with `constexpr` and `QT_TR_NOOP`

Reply:
Thank you for the explanation about C++ constructs but my comment was about readability.

Please use the following standard code for translations in UGENE:
```
QString unexpectedEnzymeTypeMessage = tr("Unexpected enzyme type: %1");
```

instead of the non standard `static constexpr const char* & QT_TR_NOOP`. 

A non-standard code stands out when you read/review the code and here there is no reason to use a non-standard solution

Reply:
OK